src/Symfony/Component/Notifier/Bridge/Novu/NovuOptions.php

```diff
@@ -26,6 +26,7 @@ public function __construct(
         private readonly string|null $phone = null,
         private readonly string|null $avatar = null,
         private readonly string|null $locale = null,
+        private readonly array $overrides = [],
```

Comment:
It should be set as the last to prevent BC break

Reply:
@stloyd  I've developed this component, which is only availible in 6.4 , (which isnt released yet) so dont think there is  any BC. and the `$overrides` is a option novu needs more than `$options`